spec/models/grda_warehouse/tasks/client_cleanup_spec.rb

```diff
@@ -247,7 +247,7 @@
     end
 
     it 'only updates SSN from clients with a value' do
-      source_1.update(SSN: @ssn1, SSNDataQuality: 99)
```

Comment:
I believe this test was specifically here to catch the situation where there's bad SSNDataQuality data, but not necessarily a missing SSN.